### PR TITLE
Zone colour proportional to percentages

### LINKED ISSUE #311: Make Zone colour proportional to percentages
One to explore: make the scale more constant, e.g. between scenarios or regions.

### PR DESCRIPTION
**Pull request by nikolai-b**
Closes #311

### CODE REVIEW

**Comment by nikolai-b** [comment]
@Robinlovelace bump....

**Comment by Robinlovelace** [comment]
Will test asap. Ta.

File: server-base.R [hunk 1]
```diff
@@ -25,7 +25,7 @@ zcols <- c("darkslategrey", "yellow")
 # expect pct-data as a sibling of pct-shiny
 data_dir_root <- file.path(shiny_root, '..', 'pct-data')
 # packages required
-cran_pkgs <- c("shiny", "RColorBrewer", "httr", "rgdal", "rgeos", "leaflet", "DT", "shinyjs")
+cran_pkgs <- c("shiny", "RColorBrewer", "httr", "rgdal", "rgeos", "leaflet", "DT", "shinyjs", "sp", "dplyr")
```
Comment (nikolai-b) [inline server-base.R]: Out of interest why?  I was working without these for me.  Also this seems unrelated to #311...

**Comment by Robinlovelace** [comment]
Because it's more explicit and causes errors on my Windows box.
